src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -270,9 +270,7 @@ void FixedwingAttitudeControl::Run()
 	perf_begin(_loop_perf);
 
 	// only run controller if attitude changed
-	vehicle_attitude_s att;
-
-	if (_att_sub.update(&att)) {
+	if (_att_sub.updated() || (hrt_elapsed_time(&_last_run) > 20_ms)) {
```

Comment:
same comment as @sfuhrer on the `20_ms` here - would be nicer defined as const in the header